src/config/sidebar.ts

```diff
@@ -974,108 +974,12 @@ export const SIDEBAR = {
-              url: "chainlink-nodes/oracle-jobs/task-types/task_cborparse",
-            },
-            {
-              title: "ETH ABI Decode",
-              url: "chainlink-nodes/oracle-jobs/task-types/task_eth_abi_decode",
-            },
-            {
-              title: "ETH ABI Decode Log",
-              url: "chainlink-nodes/oracle-jobs/task-types/task_eth_abi_decode_log",
-            },
-            {
-              title: "ETH ABI Encode",
-              url: "chainlink-nodes/oracle-jobs/task-types/task_eth_abi_encode",
-            },
-            {
-              title: "ETH Call",
-              url: "chainlink-nodes/oracle-jobs/task-types/task_eth_call",
-            },
-            {
-              title: "ETH Tx",
-              url: "chainlink-nodes/oracle-jobs/task-types/task_eth_tx",
-            },
-            {
-              title: "Multiply",
-              url: "chainlink-nodes/oracle-jobs/task-types/task_multiply",
-            },
-            {
-              title: "Divide",
-              url: "chainlink-nodes/oracle-jobs/task-types/task_divide",
-            },
-            {
-              title: "Any",
-              url: "chainlink-nodes/oracle-jobs/task-types/task_any",
-            },
-            {
-              title: "Mean",
-              url: "chainlink-nodes/oracle-jobs/task-types/task_mean",
-            },
-            {
-              title: "Median",
-              url: "chainlink-nodes/oracle-jobs/task-types/task_median",
-            },
-            {
-              title: "Mode",
-              url: "chainlink-nodes/oracle-jobs/task-types/task_mode",
-            },
-            {
-              title: "Sum",
-              url: "chainlink-nodes/oracle-jobs/task-types/task_sum",
-            },
-            {
-              title: "Less Than",
-              url: "chainlink-nodes/oracle-jobs/task-types/task_lessthan",
-            },
-            {
-              title: "Length",
-              url: "chainlink-nodes/oracle-jobs/task-types/task_length",
-            },
-            {
-              title: "Hex Decode",
-              url: "chainlink-nodes/oracle-jobs/task-types/task_hexdecode",
-            },
-            {
-              title: "Hex Encode",
-              url: "chainlink-nodes/oracle-jobs/task-types/task_hexencode",
-            },
-            {
-              title: "Base64 Decode",
-              url: "chainlink-nodes/oracle-jobs/task-types/task_base64decode",
-            },
-            {
-              title: "Base64 Encode",
-              url: "chainlink-nodes/oracle-jobs/task-types/task_base64encode",
-            },
-            {
-              title: "Uppercase",
-              url: "chainlink-nodes/oracle-jobs/task-types/task_uppercase",
-            },
-            {
-              title: "Lowercase",
-              url: "chainlink-nodes/oracle-jobs/task-types/task_lowercase",
+              title: "Pipelines and Task Types",
```

Comment:
```suggestion
              title: "Task Types",
```

Reply:
For the main "Tasks" page, change the title to "Tasks and pipelines" - to reflect how the content was shifted